fix: line breaks in multiline strings, issue 13157 (#13274)
prettier/prettier: changelog_unreleased/javascript/13274.md

> #### Fix line breaks in multiline strings aren't replaced with literalline and confuse fits (#13274 by @GlebDolzhikov)

Reviewer: ```suggestion
#### Fix inconsistent formatting for multiline strings (#13274 by @GlebDolzhikov)
```
Most users have no idea what `fits` and `literalline` are.
